### PR TITLE
Expose release notes of the proxy on the proxy

### PR DESCRIPTION
**Pull request by nodeg**
## What does this PR change?

This will expose the release notes of the proxy directly on the proxy. During testing the 4.1.14 MU @srbarrios, @vandabarata and I found out that it would be convenient to have the proxy release notes not only available on the server but also on the proxy itself.

-  `release-notes-uyuni-proxy` spec file changes. [Test builds](https://build.opensuse.org/package/show/home:dgedon:branches:systemsmanagement:Uyuni:Master/release-notes-uyuni-proxy)
- `release-notes-susemanager-proxy` spec file changes for Head.  [Test builds](https://build.suse.de/package/show/home:dgedon:branches:Devel:Galaxy:Manager:Head:ToSLE/release-notes-susemanager-proxy)
- `release-notes-susemanager-proxy` spec file changes for 4.2. [Test builds](https://build.suse.de/package/show/home:dgedon:branches:Devel:Galaxy:Manager:4.2:ToSLE/release-notes-susemanager-proxy)

TODO:

- [x] Let changelog entries be validated by someone
- [x] Create SRs in the OBS/IBS
  - [x] [Uyuni](https://build.opensuse.org/request/show/961067)
  - [x] [Head](https://build.suse.de/request/show/267344)
  - [x] [4.2](https://build.suse.de/request/show/267343)

## GUI diff

**Only on the proxy!**

Before:
image

After:
image

- [x] **DONE**

## Documentation

I am not sure if I need documentation for that since it does not really changes the GUI but only the `index.html` of the proxy.

- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- No documentation needed: only internal and user invisible changes
- Documentation issue was created: [Link for SUSE Manager contributors](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31), [Link for community contributors](https://github.com/uyuni-project/uyuni-docs/issues/new).
- (OPTIONAL) [Documentation PR](https://github.com/uyuni-project/uyuni-docs/pulls)
- [x] **DONE**

## Test coverage
- No tests:

- [x] **DONE**

## Links

Fixes https://github.com/SUSE/spacewalk/issues/17068
[Manager 4.2](https://github.com/SUSE/spacewalk/pull/17320)

- [x] **DONE**

## Changelogs

Changelog entries created in the `.changes` file in the OBS.

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [x] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: proxy/proxy/httpd-conf/spacewalk-proxy.conf [hunk 1]
```diff
@@ -20,6 +20,22 @@
     </IfVersion>
 </Directory>
 
+<Directory "/srv/www/htdocs/docs/*">
+    Options Indexes FollowSymLinks
```
Comment (mcalmer) [inline proxy/proxy/httpd-conf/spacewalk-proxy.conf:24]: Try out what happens if you manually enter in the webbrowser "/docs/en/release-notes/" as path.
With "Options Indexes" you can see the directory index. Do we want this?
The option "FollwoSymlinks should only be set if we really have to deal with symlinks. Do we in this location?

Maybe just remove the full line?

Comment (nodeg) [reply proxy/proxy/httpd-conf/spacewalk-proxy.conf:24]: This works, we see the release notes file:
image

I was not sure about this part, so I am glad you commented. I can remove the whole line. I think the release notes are always at the same location and are always named the same, so removing the options line would be a good idea.

**Review by mcalmer — APPROVED** [review]

**Comment by mcalmer** [comment]
@nodeg you changed packages which are not that often chaned. So maybe it is better to write a changelog

**Comment by nodeg** [comment]
@mcalmer I did write a changelog for the the packages I changed in the OBS/IBS (see the SRs in the 1st post). Is there another changelog I should take care of?

**Comment by mcalmer** [comment]
@nodeg yes, a changelog is needed for all packages you made changes. Here in the  PR you changed 2 packages:
- spacewalk-proxy (proxy/proxy/spacewalk-proxy.changes)
- spacewalk-proxy-html (proxy/proxy-html/spacewalk-proxy-html.changes)

The requirement is, that we need at least 1 changelogentry for every package we want to submit.
You can check now, if these both packages already contain a changelog entry which is not yet tagged.
If there is none, you better add one directly and when you add it for one, maybe you do it directly for the other one as well.

**Comment by nodeg** [comment]
I see, I have to create entries in the `.changes` files for the proxy here, too. Ok I will make another commit. Thank you @mcalmer!

**Review by mcalmer — CHANGES_REQUESTED** [review]
In this git repo, we do not create the changelog "header" . Only the line with the changes is needed.
The Header is created when we tag the package.

Please remove the header (first 3 lines) of the changes.

**Review by mcalmer — APPROVED** [review]

**Pull request merged**